wiki/Tournaments/CFC/CFIC_1/fr.md

```diff
@@ -0,0 +1,169 @@
+| **B** | ![][flag_FR] [YANOO123](/users/5122949) | ![][flag_FR] [Yruama](/users/8221467) | ![][flag_FR] [oasix42](/users/6183012) | ![][flag_FR] [rpp1](/users/3463504) |
+| **C** | ![][flag_BE] [-\[Smoked\]-](/users/8731044) | ![][flag_FR] [floflim](/users/5954015) | ![][flag_FR] [Khalemm](/users/11983416) | ![][flag_FR] [redjiii](/users/1378728) |
+| **D** | ![][flag_FR] [Realmas](/users/6567640) | ![][flag_FR] [Rizhen-](/users/6802893) | ![][flag_FR] [Sageru](/users/10769450) | ![][flag_FR] [Zard](/users/6277626) |
+
+-----------------------
+
+## Planning des matches: Phase de poules
+
+### Vendredi, 1 Mars 2019
+
+| Match ID | Joueur A |  |  | Joueur B | Horaire du match (UTC +1) |
+| :-: | --: | :-: | :-: | :-- | :-: |
+| #3 | Aequo3 | ![][flag_FR] | ![][flag_FR] | pierreCHT | **17:00** |
+
+### Samedi, 2 Mars 2019
+
+| Match ID | Joueur A |  |  | Joueur B | Horaire du match (UTC +1) |
+| :-: | --: | :-: | :-: | :-- | :-: |
+| #1 | pierreCHT | ![][flag_FR] | ![][flag_FR] | 4L-Chan 3 Porte | **13:00** |
+| #2 | ACinfr | ![][flag_FR] | ![][flag_FR] | Aequo3 | **13:30** |
+| #7 | Yruama | ![][flag_FR] | ![][flag_FR] | oasix42 | **14:00** |
+| #9 | YANOO123 | ![][flag_FR] | ![][flag_FR] | Yruama | **14:45** |
+| #18 | Khalemm | ![][flag_FR] | ![][flag_BE] | -\[Smoked\]- | **15:00** |
+| #10 | oasix42 | ![][flag_FR] | ![][flag_FR] | rpp1 | **15:15** |
+| #6 | 4L-Chan 3 Porte | ![][flag_FR] | ![][flag_FR] | Aequo3 | **15:30** |
+| #11 | rpp1 | ![][flag_FR] | ![][flag_FR] | Yruama | **15:45** |
+| #5 | ACinfr | ![][flag_FR] | ![][flag_FR] | pierreCHT | **16:00** |
+| #13 | floflim | ![][flag_FR] | ![][flag_FR] | Khalemm | **16:00** |
+| #12 | oasix42 | ![][flag_FR] | ![][flag_FR] | YANOO123 | **16:15** |
+| #14 | redjiii | ![][flag_FR] | ![][flag_BE] | -\[Smoked\]- | **16:30** |
+| #16 | Khalemm | ![][flag_FR] | ![][flag_FR] | redjiii | **17:00** |
+| #8 | rpp1 | ![][flag_FR] | ![][flag_FR] | YANOO123 | **17:30** |
+| #4 | 4L-Chan 3 Porte | ![][flag_FR] | ![][flag_FR] | ACinfr | **19:00** |
+
+### Dimanche, 3 Mars 2019
+
+| Match ID | Joueur A |  |  | Joueur B | Horaire du match (UTC +1) |
+| :-: | --: | :-: | :-: | :-- | :-: |
+| #19 | Rizhen- | ![][flag_FR] | ![][flag_FR] | Sageru | **13:00** |
+| #21 | Realmas | ![][flag_FR] | ![][flag_FR] | Rizhen- | **13:45** |
+| #24 | Sageru | ![][flag_FR] | ![][flag_FR] | Realmas | **14:30** |
+| #20 | Zard | ![][flag_FR] | ![][flag_FR] | Realmas | **15:00** |
+| #23 | Zard | ![][flag_FR] | ![][flag_FR] | Rizhen- | **16:00** |
+| #15 | -\[Smoked\]- | ![][flag_BE] | ![][flag_FR] | floflim | **17:00** |
+| #17 | redjiii | ![][flag_FR] | ![][flag_FR] | floflim | **17:30** |
+| #22 | Sageru | ![][flag_FR] | ![][flag_FR] | Zard | **18:00** |
+
+-----------------------
+
+## Liste des maps
+
+### Phase de poules
+
+**[Télécharge le mappack ici! (55 MB)](https://mega.nz/#!FotnlQzR!bTxeqfGRbm-sOboVhq-AP31WP4r2ExNaGYNwpFrCIdU)**
```

Comment:
"Téléchargez le mappack ici!" instead. The current sentence is at the first person, and makes it look like you're directly speaking to someone.